documentation/extracts.md

```diff
@@ -0,0 +1,61 @@
+
+| Type           | Purpose                                                                                                                                                           |
+|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CASE_DATA      | Provides a flat view of case data                                                                                                                                 |
+| ALLOCATIONS    | Provides a view of every time a case has been allocated and unallocated, as well as the entry and exit times for each stage and also the deadlines for each stage |
+| TOPICS         | Provides all the topics added to a case by date range                                                                                                             |
+| CORRESPONDENTS | Link the UUIDS from case data to correspondents                                                                                                                   |
+| CASE_NOTES     | Provides the case notes added to a case by date range                                                                                                             |
+| EXTENSIONS     | Provides details on all extensions applied to a case                                                                                                              |
+| APPEALS        | Provides details on all appeals added to a case                                                                                                                   |
+| INTERESTS      | Provides details on all interest from internal and external sources recorded against a case                                                                       |
+| SUSPENSIONS    | Provides details on each time a case has had a suspension applied or removed                                                                                      |
+
+### Environment 
+The available environment export types are:
+
+| Type        | Purpose                                                                  |
+|-------------|--------------------------------------------------------------------------|
+| Topics      | Provides a list of topics in the database, with matching UUIDS           |
+| Teams       | Provides a list of teams in the database, with matching UUIDS            |
+| Units/teams | Provides a list of units and their associated teams, with matching UUIDS |
+| Users       | Provides a list of users in the database, with matching UUIDS            |
+
+### Single Object, Multiple Use (SOMU)
+Only some combinations of case types and SOMU types result in valid extract URLs, as not all case types use SOMU functionality.
+
+## Permissions
+Users do not get access to the extract functionality by default. A subset of users can access extracts, providing they have the appropriate Keycloak roles.
```

Comment:
Should the X-Auth-Group(s) be listed here?

Reply:
I see this as configuration rather than feature documentation

Reply:
Can there be a link to where to look this up instead?

Reply:
I've added some extra context around this, if you'd like to have another look and see if the added detail is more clear now please?

Reply:
Yes, thanks.